tests/Resource/CloudAccount/EndpointTest.php

```diff
@@ -333,4 +343,106 @@ public function testSetPhpVersion() {
           ->setPhpVersion($entity, '7.2');
       });
   }
+
+  /**
+   * @covers Endpoint::createBackup
+   */
+  public function testCreateBackup() {
+    // custom request handler for sandbox
+    $request_handler = function ($request, $options) {
+      // check request path
+      $this->assertEquals('cloud-account/1/backup', $request->getUri()->getPath());
+
+
+      return new GuzzleResponse(
+        200,
+        ['Content-type' => 'application/json'],
+        $this->_getResource(static::_RESOURCE_NEW_BACKUP, false)
+      );
+    };
+
+    // kick off
+    $this->_getSandbox(null, $request_handler)
+      ->play(function ($api, $sandbox) {
+        $entity = $api->getModel(static::_SUBJECT_MODEL_FQCN)->set('id',1);
+        $endpoint = $api->getEndpoint(static::_SUBJECT_MODULE);
+        $results = $endpoint->createBackup($entity);
+        $this->assertEquals('filename.tgz',$results->get('filename'));
+        $this->assertEquals("123 MB",$results->get('filesize'));
+        $this->assertEquals(456,$results->get('filesize_bytes'));
+      });
+  }
+
+  /**
+   * @covers Endpoint::downloadBackup
+   */
+  public function testDownloadBackup() {
+    // custom request handler for sandbox
+    $assertionCounter = 0;
+    $request_handler = function ($request, $options) use (&$assertionCounter) {
+      // check request path
+
+      switch (++$assertionCounter) {
+        case 1:
+          $this->assertEquals('cloud-account/1/backup', $request->getUri()->getPath());
+          break;
+
+        case 2:
+          $this->assertEquals('/siteworx/index', $request->getUri()->getPath());
+          break;
```

Comment:
should have a `default` case that does something like `$this->fail("unexpected request: {$request->getUri()->getPath()}")`